packages/wonder-blocks-popover/src/components/__tests__/popover.test.tsx

```diff
@@ -502,6 +501,87 @@ describe("Popover", () => {
                 }),
             ).toHaveFocus();
         });
+
+        it("should close the popover when the user tabs before the trigger element", async () => {
+            // Arrange
+            render(
+                <Popover
+                    dismissEnabled={true}
+                    placement="top"
+                    content={
+                        <PopoverContent
+                            title="Title"
+                            content="content"
+                            closeButtonVisible={true}
+                        />
+                    }
+                >
+                    {({open}: any) => (
+                        <button data-anchor onClick={open}>
+                            Open default popover
+                        </button>
+                    )}
+                </Popover>,
+            );
+
+            // open the popover
+            await userEvent.click(
+                await screen.findByRole("button", {
+                    name: "Open default popover",
+                }),
+            );
+
+            // Act
+            // Focus on the reference element
+            await userEvent.tab({shift: true});
+            // Focus on the previous element before the popover (triggers
+            // closing the popover)
+            await userEvent.tab({shift: true});
+
+            // Assert
+            await waitFor(() => {
+                expect(screen.queryByText("Title")).not.toBeInTheDocument();
+            });
+        });
+
+        it("should close the popover when the user tabs after the last focusable element inside the popover", async () => {
+            // Arrange
+            render(
+                <div>
+                    <Popover
+                        dismissEnabled={true}
+                        placement="top"
+                        content={
+                            <PopoverContent
+                                title="Title"
+                                content="content"
+                                closeButtonVisible={true}
+                            />
+                        }
+                    >
+                        <Button>Open default popover</Button>
+                    </Popover>
+                    <Button>Next button outside</Button>
+                </div>,
+            );
+
+            // open the popover
+            // open the popover by focusing on the trigger element
+            await userEvent.click(
+                await screen.findByRole("button", {
+                    name: "Open default popover",
+                }),
+            );
+
+            // Focus on the last focusable element inside the popover (dismiss button)
+            await userEvent.tab();
+
+            // Focus on the next element after the popover
+            await userEvent.tab();
+
+            // Assert
+            expect(screen.queryByText("Title")).not.toBeInTheDocument();
+        });
```

Comment:
Should we have tests for when `dismissEnabled={false}`, the popover isn't closed on focus out?